scripts/cli.py

```diff
@@ -207,7 +191,10 @@ def get_pub_addr(cloud: Optional[str], controller: str, model: str):
     if cloud == 'microk8s':
         return 'localhost'
     else:
-        return f'{get_pub_ip(controller, model)}.xip.io'
+        try:
+            return f'{get_pub_ip(controller, model)}.xip.io'
```

Comment:
Doesn't seem like `model` is very useful here. `get_pub_ip` only ever checks worker status in the `default` model now.